ecs_fargate/README.md

```diff
@@ -128,55 +177,64 @@ aws ecs run-task --cluster <CLUSTER_NAME> \
+      #(...)
+```
 
+For more information on CloudFormation templating and syntax, see the [AWS CloudFormation ECS service documentation][1].
+
+[1]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html
+{{% /tab %}}
+
+{{< /tabs >}}
 ### Metric collection
 
-After the Datadog Agent is setup as described above, the [ecs_fargate check][16] collects metrics with autodiscovery enabled. Add Docker labels to your other containers in the same task to collect additional metrics.
+After the Datadog Agent is setup as described above, the [ecs_fargate check][11] collects metrics with autodiscovery enabled. Add Docker labels to your other containers in the same task to collect additional metrics.
 
-For details on collecting integration metrics, see [Integration Setup for ECS Fargate][8].
+For details on collecting integration metrics, see [Integration Setup for ECS Fargate][12].
 
 #### DogStatsD
 
-Metrics are collected with [DogStatsD][17] through UDP port 8125.
+Metrics are collected with [DogStatsD][13] through UDP port 8125.
 
 To send custom metrics by listening to DogStatsD packets from other containers, set the environment variable `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` to `true` within the Datadog Agent container.
 
 #### Other environment variables
 
-For environment variables available with the Docker Agent container, see the [Docker Agent][18] page. **Note**: Some variables are not be available for Fargate.
+For environment variables available with the Docker Agent container, see the [Docker Agent][14] page. **Note**: Some variables are not be available for Fargate.
 
 
 | Environment Variable               | Description                                    |
 |------------------------------------|------------------------------------------------|
 | `DD_DOCKER_LABELS_AS_TAGS`         | Extract docker container labels                |
-| `DD_CONTAINER_LABELS_AS_TAGS`      | Extract container labels                       |
-| `DD_DOCKER_ENV_AS_TAGS`            | Extract docker container environment variables |
-| `DD_CONTAINER_ENV_AS_TAGS`         | Extract container environment variables        |
-| `DD_KUBERNETES_POD_LABELS_AS_TAGS` | Extract pod labels                             |
 | `DD_CHECKS_TAG_CARDINALITY`        | Add tags to check metrics                      |
 | `DD_DOGSTATSD_TAG_CARDINALITY`     | Add tags to custom metrics                     |
 
-For global tagging, it is recommended to use `DD_CONTAINER_LABELS_AS_TAGS`. With this method, the Agent pulls in tags from your container labels. This requires you to add the appropriate labels to your other containers. Labels can be added directly in the [task definition][19].
+For global tagging, it is recommended to use `DD_DOCKER_LABELS_AS_TAGS`. With this method, the Agent pulls in tags from your container labels. This requires you to add the appropriate labels to your other containers. Labels can be added directly in the [task definition][15].
```

Comment:
let's keep `dd_container_labels_as_tags`, per the [PR](https://github.com/DataDog/datadog-agent/pull/9542), you can see that it's more generic than just docker. So if containerd is used, it will work oob.

Reply:
I think that was the goal with that PR. But with Fargate right now from my testing, only `DD_DOCKER_ENV_AS_TAGS` currently works. I thought we had an escalation somewhere to track this, but doesn't look like it came through. I'll raise something for this.